bigchaindb/common/exceptions.py

```diff
@@ -102,3 +102,19 @@ class GenesisBlockAlreadyExistsError(ValidationError):
 
 class MultipleValidatorOperationError(ValidationError):
     """Raised when a validator update pending but new request is submited"""
+
+
+class MultipleInputsError(ValidationError):
+    """Raised if there were multiple inputs when only one was expected"""
+
+
+class InvalidProposer(ValidationError):
+    """Raised if the public key is not a part of the validator set"""
+
+
+class UnequalValidatorSet(ValidationError):
```

Comment:
nitpick: `InvalidValidatorSet` makes more sense for this exception?

Reply:
I thought about that name `InvalidValidatorSet`, but the code [here](https://github.com/bigchaindb/bigchaindb/pull/2392/files#diff-dff01128fef040477df965185b717a15R119) tries to detect if the validator sets are the same or different. The only thing we wish to imply is their inequality and not validity.